spec/spec.md

```diff
@@ -836,7 +867,316 @@ processing-related rules above:
 
-### Input Evaluation
+
+## Data Minimization Feature
+
+The Data Minimization [[ref:Feature]] introduces properties enabling [[ref:Verifier]]
+to request that processing entities minimize returned data, by limiting
+submitted fields to those requested or applying a predicate and returning the
+result.
+
+### Requiring Limited Disclosure
+
+The Data Minimization [[ref:Feature]] extends the [[ref:Input Descriptor Object]] 
+`constraints` object to add a `limit_disclosure` property.
+
+When using this [[ref:Feature]], the _constraints object_ ****MAY**** contain a
+`limit_disclosure` property. If present, its value ****MUST**** be one of the
+ following strings:
+- `preferred` - This indicates that the processing entity ****SHOULD****
+  limit submitted fields to those listed in the `fields` array (if
+  present).
+- `required` - This indicates that the processing entity ****MUST****
+  limit submitted fields to those listed in the `fields` array (if
+  present).
+
+Omission of the `limit_disclosure` property indicates the processing
+entity ****MAY**** submit a response that contains more than the data
+described in the `fields` array.
+
+### Applying a predicate
+
+The data minimization [[ref:Feature]] extends the [[ref:Input Descriptor Object]]
+`constraints.fields` object to add a `predicate` property.
+
+When using this [[ref:Feature]], the _fields object_ ****MAY**** contain a 
+[`predicate` property](#predicate-property). If the `predicate` property is 
+present, the `filter` property ****MUST**** also be present.
+
+:::note The inclusion of the `predicate` property indicates that the
+processing entity returns a boolean, rather than a value returned
+from evaluation of the
+[JSONPath](https://goessner.net/articles/JsonPath/) string
+expressions in the `path` array. The boolean returned is the result
+of using the `filter` property's
+[JSON Schema](https://json-schema.org/specification.html)
+descriptors against the evaluated value. Exclusion of the `predicate`
+property indicates that the processing entity returns the value
+returned from evaluation of the
+[JSONPath](https://goessner.net/articles/JsonPath/) string
+expressions in the `path` array.
+:::
+
+The value of `predicate` ****MUST**** be one of the following strings:
+- `required` - This indicates that the returned value ****MUST****
+  be the boolean result of applying the value of the `filter`
+  property to the result of evaluating the `path` property.
+  :::note Using a value of `required` for the `predicate` property
+  may severely limit the responses a [[ref:Holder]] may be able to
+  make. Many signature schemes do not support deriving predicates,
+  even those signature schemes which are otherwise ZKP-capable. A
+  [[ref:Verifier]] should be sure they support such schemes, and
+  have high confidence they are also supported by the
+  [[ref:Holder]], before indicating predicate responses are
+  required.:::
+- `preferred` - This indicates that the returned value
```

Comment:
#supernit maybe the order of `"preferred"` and `"required"` should be the same for properties that follow that pattern?

Reply:
that's not a supernit that's a valid question, i'd say

Reply:
Yeah, i'm putting this on hold for now